wondrous-app/components/Common/MilestoneTask/index.tsx

```diff
@@ -1 +1,58 @@
-export { default } from './MilestoneTasks';
+import { useQuery } from '@apollo/client';
+import { GET_TASKS_FOR_MILESTONE } from 'graphql/queries';
+import { useEffect, useState } from 'react';
+import { useInView } from 'react-intersection-observer';
+import styled from 'styled-components';
+import MilestoneTaskFilter from './MilestoneTaskFilter';
+import MilestoneTaskList from './MilestoneTaskList';
+import MilestoneTasksCreate from './MilestoneTasksCreate';
+
+export const LoadMore = styled.div`
+  height: 10px;
+  display: ${(props) => (props.hasMore ? 'block' : 'none')};
+`;
+
+const getDataLength = (data) => data?.getTasksForMilestone?.length;
+
+const useGetTasksForMilestone = ({ milestone, status }) => {
+  const { id } = milestone;
+  const limit = 10;
```

Comment:
we can use LIMIT from services/board. It's set to 10 but in the future if we want to change it's easier to do it from one place 

`import { LIMIT } from 'services/board'`